lib/cosmos/script/limits.rb

```diff
@@ -98,6 +98,10 @@ def get_limits_event(id, non_block = false)
         if result[0] == :LIMITS_CHANGE
           result[1][3] = result[1][3].to_s.intern if result[1][3]
           result[1][4] = result[1][4].to_s.intern if result[1][4]
+          if result[1][5] and result[1][6]
+            result[1][5] = Time.at(result[1][5], result[1][6]).sys
+            result[1].delete_at(6)
+          end
```

Comment:
What is this doing?

Reply:
This is the packet_time of the packet with the limits violation.